lib/presentation/widgets/sign_in_form.dart

```diff
@@ -44,6 +55,11 @@ class _MyLogFormWidgetState extends State<SignInForm> {
             labelText: LocaleKeys.enter_password.tr(),
             validator: FormValidators.passwordRegValidator,
             controller: _pass,
+            obscureText: !secureController,
```

Comment:
shouldnt it be `obscureText: secureController`? So text is obscruted when controller is secured?